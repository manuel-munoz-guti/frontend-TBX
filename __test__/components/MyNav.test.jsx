const { render, screen } = require("@testing-library/react");
const { MyNav } = require("../../src/components/MyNav");

describe('Test on component MyNav', () => {
    test('Should match with the snapshost', () => {
        const { container } = render(<MyNav/>);
        expect(container).toMatchSnapshot(); 
    });
    test('Should the header will have the text', () => {
        render(<MyNav />);
        expect(screen.getByText('REACT CHALLENGE - TBX'));
    });
});