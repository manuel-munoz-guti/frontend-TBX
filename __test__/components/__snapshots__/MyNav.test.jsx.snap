// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test on component MyNav Should match with the snapshost 1`] = `
<div>
  <nav
    class="navbar navbar-expand navbar-dark bg-dark"
  >
    <div
      class="container"
    >
      <a
        class="navbar-brand"
        href="#home"
      >
        <img
          alt=""
          class="d-inline-block align-top"
          height="30"
          src="../../src/favicon.svg"
          width="30"
        />
          
        REACT CHALLENGE - TBX
      </a>
    </div>
  </nav>
</div>
`;
